requirements.txt

```diff
@@ -1,48 +1,5 @@
-attrs==18.1.0
-cachetools==3.1.0
-certifi==2018.11.29
-chardet==3.0.4
-click==6.7
-cymem==1.31.2
-cytoolz==0.9.0.1
-dill==0.2.7.1
-elasticsearch==6.3.1
-elasticsearch-dsl==6.1.0
-Faker==0.8.15
-google-api-core==1.7.0
-google-auth==1.6.3
-google-cloud-core==0.29.1
-google-cloud-storage==1.14.0
-google-resumable-media==0.3.2
-googleapis-common-protos==1.5.8
-idna==2.8
-ipaddress==1.0.22
-more-itertools==4.1.0
-msgpack==0.6.1
-msgpack-numpy==0.4.1
-msgpack-python==0.5.6
-murmurhash==0.28.0
-numpy==1.14.3
-pathlib==1.0.1
-plac==0.9.6
-pluggy==0.6.0
-preshed==1.0.1
-protobuf==3.6.1
-py==1.5.3
-pyasn1==0.4.5
-pyasn1-modules==0.2.4
-pytest==3.5.1
-python-dateutil==2.7.3
-pytz==2018.9
-regex==2017.4.5
-requests==2.21.0
-rsa==4.0
-six==1.11.0
-termcolor==1.1.0
-text-unidecode==1.2
-thinc==6.10.3
-toolz==0.9.0
-tqdm==4.23.4
-ujson==1.35
-urllib3==1.24.2
-wrapt==1.10.11
+urllib3>=1.26,<2
```

Comment:
This `requirements.txt` asks for a lot more dependencies than are actually used by `anonymize-it`. To make installation quicker and not require binary compilation for numpy, etc I reduced it to only required dependencies.